UnityProject/Assets/Scripts/Items/Cargo/MiningScanner.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections;
+using Core.Lighting;
+using Mirror;
+using UnityEngine;
+
+namespace Items.Cargo
+{
+	public class MiningScanner : NetworkBehaviour, IInteractable<HandActivate>
+	{
+
+		public bool IsAdvanced = false;
+		public float NormalScanCooldown = 10f;
+		public float AdvancedScanCooldown = 4f;
+
+		private bool isOnCooldown = false;
+
+
+		[TargetRpc]
+		public void TargetStartScanning(NetworkConnection target)
+		{
+			Debug.Log("scanning");
```

Comment:
Can this be removed?

Reply:
I'd like to leave this here because I still want to do some tests when this gets merged on staging